benchmarks/sequel/benchmark.rb

```diff
@@ -46,20 +46,21 @@ class Post < Sequel::Model
 end
 
 10000.times {
-  Post.create(title: Random.alphanumeric(30),
-              type_name: Random.alphanumeric(10),
-              key: Random.alphanumeric(10),
-              body: Random.alphanumeric(100),
-              upvotes: rand(30),
-              author_id: rand(30))
+  DB[:posts].insert(
+    title: Random.alphanumeric(30),
+    type_name: Random.alphanumeric(10),
+    key: Random.alphanumeric(10),
+    body: Random.alphanumeric(100),
+    upvotes: rand(30),
+    author_id: rand(30))
 }
```

Comment:
I don't think we should change the initialization code since this is less idiomatic, but I will merge the change to the lookup code.

Reply:
OK, I'll amend the commit to not include that change.